pandas/tests/io/json/test_json_table_schema_ext_dtype.py

```diff
@@ -273,3 +277,18 @@ def test_to_json(self, df):
         expected = OrderedDict([("schema", schema), ("data", data)])
 
         assert result == expected
+
+    def test_json_ext_dtype_reading(self):
+        # GH#40255
+        df = DataFrame(
+            {
+                "a": Series([2, NA], dtype="Int64"),
+                "b": Series([1.5, NA], dtype="Float64"),
+                "c": Series([True, NA], dtype="boolean"),
+            },
+            index=Index([1, NA], dtype="Int64"),
+        )
+        expected = df.copy()
+        data_json = df.to_json(orient="table", indent=4)
+        result = read_json(data_json, orient="table")
+        tm.assert_frame_equal(result, expected)
```

Comment:
* IMO there should be a test with only `read_json` instead of doing roundtrip
* Is there a reason the roundtrip test is different from the OP one?

Reply:
Can add one.

Yes, the op only testet Int64, but we have different cases for Int64/Float64 and Boolean, so we should cover every case

Reply:
added the test